pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -83,18 +81,7 @@ def __init__(
             cluster_environment=cluster_environment,
             checkpoint_io=checkpoint_io,
         )
-        if num_nodes is not None:
-            rank_zero_deprecation(
-                "Argument `num_nodes` in `DDPSpawnPlugin` is deprecated in v1.4, and will be removed in v1.6. "
-                "Notice that it will be overriden by the trainer setting."
-            )
-        self._num_nodes = num_nodes or 1
```

Comment:
this line needs to stay and be modified (see setter below) to 

`self._num_nodes = 1`